### ISSUE TITLE
'make clean' and 'make check' errors on 'insensitive.sh' (executable bit not set ?)

### ISSUE THREAD

**Issue by mhoes** [linux-test-project/lcov#207]
Hi,

When running 'make clean', I get this error "`./insensitive.sh: Permission denied`" :

```
$ make clean
rm -f lcov-*.tar.gz
rm -f lcov-*.rpm
make -C example clean
make[1]: Entering directory '/home/maarten/src/lcov/example'
rm -rf *.o *.bb *.bbg *.da *.gcno *.gcda *.info output example \
descriptions
make[1]: Leaving directory '/home/maarten/src/lcov/example'
make -C tests -s clean
  CLEAN   tests
  CLEAN   tests/genhtml
  CLEAN   tests/lcov
  CLEAN   tests/lcov/add
  CLEAN   tests/lcov/diff
  CLEAN   tests/lcov/diff/old
  CLEAN   tests/lcov/diff/new
  CLEAN   tests/lcov/misc
  CLEAN   tests/lcov/summary
  CLEAN   tests/lcov/extract
  CLEAN   tests/lcov/demangle
  CLEAN   tests/lcov/exception
  CLEAN   tests/gendiffcov
  CLEAN   tests/gendiffcov/simple
  CLEAN   tests/gendiffcov/filter
  CLEAN   tests/gendiffcov/function
  CLEAN   tests/gendiffcov/insensitive
make[3]: ./insensitive.sh: Permission denied
  CLEAN   tests/py2lcov
find . -name '*.tdy' -o -name '*.orig' | xargs rm -f
```

And the same thing with 'make check'
```
$ make check
<-- snip -- >
gendiffcov/simple/script.sh ....... [pass] (time 14.0s, mem 60.2MB)
gendiffcov/filter/filter.pl ....... [pass] (time 0.0s, mem 13.6MB)
gendiffcov/function/function.sh ... [pass] (time 4.6s, mem 60.3MB)
gendiffcov/insensitive/insensitive. [fail] (time 0.0s, mem 2.8MB)
  COMMAND ....: "/home/maarten/src/lcov/tests/gendiffcov/insensitive/insensitive.sh  *"
  OUTPUT .....:
    bash: line 1: /home/maarten/src/lcov/tests/gendiffcov/insensitive/insensitive.sh: Permission denied
  EXITCODE ...: 126
32 tests executed, 31 passed, 1 failed, 0 skipped (time 34.3s, mem 956.2MB)
Result log stored in /home/maarten/src/lcov/tests/test.log
if [ "x" != 'x' ] ; then \
  ( cd tests ; cover ) ; \
fi
```

I'm guessing it was intended to set the execute bit for that file, but I'll leave that up to you. ;)

**Comment by mhoes** [comment]
And it looks like the same is true for `'./tests/gendiffcov/insensitive/version.sh'`.

**Comment by henry2cox** [comment]
yeah - all 3 test scripts are missing the execute bit.
My bad.
I'll push a fix later this afternoon or tomorrow.

**Comment by henry2cox** [comment]
should be fixed in  38e39e74db14836

Sorry about that :-(
I would have sworn that I had asked git to set the execute bit.  Guess not though.

**Comment by mhoes** [comment]
Thanks.
